### ISSUE TITLE
Build of v3.9.0 fails with `--feature=rkyv`

### ISSUE THREAD

**Issue by blp** [reem/rust-ordered-float#137]
v3.9.0 uses the `rend` feature in `rkyv` that `rkyv` only implements conditionally on other features. This breaks the build if you specify the `rkyv` feature. This is true in v3.9.0 but not in v3.8.0:

```sh
[blp@sigxcpu rust-ordered-float]$ git checkout v3.8.0
HEAD is now at 651ead6 Version 3.8.0
[blp@sigxcpu rust-ordered-float]$ cargo build --features=rkyv
    Finished dev [unoptimized + debuginfo] target(s) in 0.01s
[blp@sigxcpu rust-ordered-float]$ git checkout v3.9.0
Previous HEAD position was 651ead6 Version 3.8.0
HEAD is now at 2650e6e Version 3.9.0
[blp@sigxcpu rust-ordered-float]$ cargo build --features=rkyv
   Compiling ordered-float v3.9.0 (/home/blp/feldera/rust-ordered-float)
error[E0433]: failed to resolve: could not find `rend` in `rkyv`
    --> src/lib.rs:2006:45
     |
2006 |     rkyv_eq_ord! { OrderedFloat, f32, rkyv::rend::f32_le }
     |                                             ^^^^ could not find `rend` in `rkyv`

error[E0433]: failed to resolve: could not find `rend` in `rkyv`
    --> src/lib.rs:2007:45
     |
2007 |     rkyv_eq_ord! { OrderedFloat, f32, rkyv::rend::f32_be }
     |                                             ^^^^ could not find `rend` in `rkyv`

error[E0433]: failed to resolve: could not find `rend` in `rkyv`
    --> src/lib.rs:2008:45
     |
2008 |     rkyv_eq_ord! { OrderedFloat, f64, rkyv::rend::f64_le }
     |                                             ^^^^ could not find `rend` in `rkyv`

error[E0433]: failed to resolve: could not find `rend` in `rkyv`
    --> src/lib.rs:2009:45
     |
2009 |     rkyv_eq_ord! { OrderedFloat, f64, rkyv::rend::f64_be }
     |                                             ^^^^ could not find `rend` in `rkyv`

error[E0433]: failed to resolve: could not find `rend` in `rkyv`
    --> src/lib.rs:2010:39
     |
2010 |     rkyv_eq_ord! { NotNan, f32, rkyv::rend::f32_le }
     |                                       ^^^^ could not find `rend` in `rkyv`

error[E0433]: failed to resolve: could not find `rend` in `rkyv`
    --> src/lib.rs:2011:39
     |
2011 |     rkyv_eq_ord! { NotNan, f32, rkyv::rend::f32_be }
     |                                       ^^^^ could not find `rend` in `rkyv`

error[E0433]: failed to resolve: could not find `rend` in `rkyv`
    --> src/lib.rs:2012:39
     |
2012 |     rkyv_eq_ord! { NotNan, f64, rkyv::rend::f64_le }
     |                                       ^^^^ could not find `rend` in `rkyv`

error[E0433]: failed to resolve: could not find `rend` in `rkyv`
    --> src/lib.rs:2013:39
     |
2013 |     rkyv_eq_ord! { NotNan, f64, rkyv::rend::f64_be }
     |                                       ^^^^ could not find `rend` in `rkyv`

For more information about this error, try `rustc --explain E0433`.
error: could not compile `ordered-float` (lib) due to 8 previous errors
[blp@sigxcpu rust-ordered-float]$ 
```

**Comment by mbrubeck** [comment]
cc #136

**Comment by mbrubeck** [comment]
Fixed in ordered-float 3.9.1.

**Comment by blp** [comment]
> Fixed in ordered-float 3.9.1.

Beautiful! Thank you!
